src/plugins/intel_gna/src/gna_graph_tools.hpp

```diff
@@ -173,12 +173,12 @@ inline std::pair<InferenceEngine::CNNLayerPtr, std::vector<int>> CNNNetCheckNext
     int iidx,
```

Comment:
oidx, iidx, new_oidx  and new_iidx are compared everywhere to size_t  from  .size() of containers.
it they cannot have negavative value maybe it would be good to change their type to size_t?

Reply:
oidx, iidx are arguments to the function, so I'd need to refactor a lot of other functions from gna_graph_tools.hpp, which is what we wanted to avoid in this PR - maybe something for the future